src/Stateless/Graph/IGraphStyle.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Stateless.Graph
+{
+    /// <summary>
+    /// Style definition for DotGraphFormatter
+    /// </summary>
+    public abstract class IGraphStyle
+    {
+        /// <summary>
+        /// Get the text that starts a new graph
+        /// </summary>
+        /// <returns></returns>
+        abstract internal string GetPrefix();
+
+        /// <summary>
+        /// Generate the text for a single state
+        /// </summary>
+        /// <param name="state">The state to generate text for</param>
+        /// <returns></returns>
+        abstract internal string FormatOneState(Stateless.Graph.State state);
+
```

Comment:
This interface might be used by many different implementers of formatters, so could you add more comments/documentation to guide them in their quest?